js/foundation.sticky.js

```diff
@@ -138,6 +140,32 @@ class Sticky {
                        }
                      });
     });
+	
+	this.$element.on('mutateme.zf.trigger', function (e, el) {
+		_this._setSizes(function () {
+			_this._calc(false);
+			if (_this.canStick) {
+			  if (!_this.isOn) {
+				_this._events(id);
+			  }
+			} else if (_this.isOn) {
+			  _this._pauseListeners(scrollListener);
+			}
+	  });
+	});
+	
+	this.$anchor.on('mutateme.zf.trigger', function (e, el) {
+		_this._setSizes(function () {
+			_this._calc(false);
+			if (_this.canStick) {
+			  if (!_this.isOn) {
+				_this._events(id);
+			  }
+			} else if (_this.isOn) {
+			  _this._pauseListeners(scrollListener);
+			}
+		});
+	});
```

Comment:
@coreysyms are you going to have time to look at this in the next week or two? Would like to get this in for 6.3.1